.circleci/config.yml

```diff
@@ -48,13 +48,22 @@ commands:
             echo 'export NVM_DIR="/opt/circleci/.nvm"' >> $BASH_ENV
             echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> $BASH_ENV
             echo 'nvm use 12' >> $BASH_ENV
+  pip_install_requests:
+    description: Install requests library
+    steps:
+      - run:
+          command: |
+            sudo apt update
+            sudo apt install python3-requests
```

Comment:
do we not have ```pip``` there?

Reply:
There is some sort of ssl problem using requests installed with pip and python2. That  problem is fixed in python3 so to be able to use requests with python3 this was the way it had to be installed after lots of trial end error. Hope it makes sense? This is the slack thread with all the detail: https://snyk.slack.com/archives/CDSMEJ29E/p1586268718117900

Reply:
The name of the job is probably misleading though: `pip_install_requests` but doesn't use pip to install! Maybe a more generic `install_requests_python`. Or alternatively the very specific `apt_install_python3_requests` 🤷‍♂ Up to you!

Reply:
you're right! I've changed how I install requests, but didn't change the name of the job 👍